web/themes/custom/cloudy/templates/field/field--bundle-field--group.html.twig

```diff
@@ -0,0 +1,4 @@
+{% include 'field.html.twig' with {
+  label_display: 'visually_hidden',
```

Comment:
Does this assume we never want to have a field group that displays a label? Would we just override that specific field group if we did?